setup.py

```diff
@@ -16,9 +16,16 @@ def read(*parts):
     with codecs.open(os.path.join(HERE, *parts), "rb", "utf-8") as f:
         return f.read()
 
+
+def req_file(filename):
+    with open(filename) as f:
+        content = f.readlines()
+    return [x.strip() for x in content]
+
+
 setup(
-    name='aioriak',
-    version='0.1.7',
+    name='ppl_aioriak',
```

Comment:
We should probably revert this back to `aioriak`?

Reply:
There is already 'ppl-aioriak' 1.0.0 at codeartifact, so there I don't see a problem to have 2.0.0